shopping_mall/backend/app/models/faq_category.py

```
@@ -0,0 +1,51 @@
+
+    # ── 운영 상태 ─────────────────────────────────────────────────────────────
+    is_active: Mapped[bool] = mapped_column(Boolean, default=True, nullable=False)
+
+    # ── 타임스탬프 ────────────────────────────────────────────────────────────
+    created_at: Mapped[datetime] = mapped_column(DateTime, server_default=func.now())
+    updated_at: Mapped[datetime] = mapped_column(
+        DateTime, server_default=func.now(), onupdate=func.now()
+    )
+
+    # ── 관계 ──────────────────────────────────────────────────────────────────
+    docs: Mapped[list["FaqDoc"]] = relationship(  # noqa: F821
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)